networkx/readwrite/tests/test_graph6.py

```diff
@@ -109,9 +109,59 @@ def test_write_path(self):
             assert f.read() == b">>graph6<<?\n"
 
     def test_relabeling(self):
-        G = nx.Graph([(0, 1)])
-        assert g6.to_graph6_bytes(G) == b">>graph6<<A_\n"
-        G = nx.Graph([(1, 2)])
-        assert g6.to_graph6_bytes(G) == b">>graph6<<A_\n"
-        G = nx.Graph([(1, 42)])
-        assert g6.to_graph6_bytes(G) == b">>graph6<<A_\n"
+        for edge in [(0, 1), (1, 2), (1, 42)]:
+            G = nx.Graph([edge])
+            f = BytesIO()
+            nx.write_graph6(G, f)
+            f.seek(0)
+            assert f.read() == b">>graph6<<A_\n"
+
+
+class TestToGraph6Bytes:
+    def test_null_graph(self):
+        G = nx.null_graph()
+        assert g6.to_graph6_bytes(G) == b">>graph6<<?\n"
+
+    def test_trivial_graph(self):
+        G = nx.trivial_graph()
+        assert g6.to_graph6_bytes(G) == b">>graph6<<@\n"
+
+    def test_complete_graph(self):
+        assert g6.to_graph6_bytes(nx.complete_graph(4)) == b">>graph6<<C~\n"
+
+    def test_large_complete_graph(self):
+        G = nx.complete_graph(67)
+        assert g6.to_graph6_bytes(G, header=False) == b"~?@B" + b"~" * 368 + b"w\n"
+
+    def test_no_header(self):
+        G = nx.complete_graph(4)
+        assert g6.to_graph6_bytes(G, header=False) == b"C~\n"
+
+    def test_complete_bipartite_graph(self):
+        G = nx.complete_bipartite_graph(6, 9)
+        assert g6.to_graph6_bytes(G, header=False) == b"N??F~z{~Fw^_~?~?^_?\n"
+
+    def test_no_directed_graphs(self):
+        G = nx.DiGraph()
+        with pytest.raises(nx.NetworkXNotImplemented):
+            g6.to_graph6_bytes(G)
```

Comment:
Might as well make this work for multigraphs too:

```suggestion

    @pytest.mark.parametrize("G", (nx.MultiGraph(), nx.DiGraph()))
    def test_no_directed_or_multi_graphs(self, G):
        with pytest.raises(nx.NetworkXNotImplemented):
            g6.to_graph6_bytes(G)
```